revocation/method.go

```diff
@@ -14,25 +14,28 @@
 package revocation
 
 import (
-	"github.com/notaryproject/notation-core-go/revocation/internal/crl"
-	"github.com/notaryproject/notation-core-go/revocation/internal/ocsp"
+	internalrevocation "github.com/notaryproject/notation-core-go/revocation/internal/revocation"
```

Comment:
In `revocation`, you can call `revocation`. There is no naming conflict.
```suggestion
	"github.com/notaryproject/notation-core-go/revocation/internal/revocation"
```

Reply:
![image](https://github.com/user-attachments/assets/5151b531-81b5-4624-befa-ef278f123275)

I tried but it doesn't work.

Reply:
```go
// revocation is an internal struct used for revocation checking
type revocation struct {
	ocspHTTPClient   *http.Client
	crlHTTPClient    *http.Client
	certChainPurpose purpose.Purpose
}
```

We have a struct `revocation` in `revocation.go`, which conflicts with this name.

Reply:
Refactored and moved the `revocation.Method` to `result.RevocationMethod`